pilot/pkg/security/authz/logger/logger.go

```diff
@@ -0,0 +1,33 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logger
+
+import (
+	"istio.io/pkg/log"
+)
+
+var (
+	authzLog = log.RegisterScope("authorization", "Istio Authorization Policy", 0)
+)
+
+func Debugf(template string, args ...interface{}) {
```

Comment:
you should wrap the logger with your own struct so that you can keep the state about where this logger is created.